README.md

```diff
@@ -12,6 +12,7 @@ This project hosts the Java client library for the Google Ads API.
 
   * Java 1.8+
   * Maven 3.0+
+  * Protobuf 3.12.2+
```

Comment:
Are you also seeing gRPC dependencies as an issue, or is protobuf by far the most common problem?

Reply:
I'm not aware of a specific lower bound for gRPC. guava is a more common source of issues. I'll add this to the line up as well.

Reply:
I'm not trying to complicate things (honest!), but would it be better if this was a separate section, or a "Dependencies" subsection of "Requirements"?

The protobuf and guava dependency requirements are different from the Java runtime requirement. The former only come into play if the user _already_ includes protobuf or guava as a transitive or direct dependency in their project. If not, google-ads-java will work just fine. In contrast, if the user tries to use Java 1.5 :tiger: , it's likely they'll run into issues.

Perhaps something like:

```
## Compatible dependencies

*  protobuf 3.12.2+
*  guava x.y.z+
```

Reply:
Sounds good. Updated.